reference/conanfile/methods.rst

```diff
@@ -1497,9 +1494,10 @@ current folder (the one containing the *conanfile.py*). The ``dst`` is relative
 generate()
 ----------
 
-.. warning::
+.. important::
```

Comment:
``generate`` as a method can be considered now stable enough, we won't break that.

Reply:
👍